eesp-ikev2.org

```diff
@@ -289,7 +289,28 @@ numbers are in use.  For a given Sub SA key, sequence numbers MUST
 remain unique and monotonically increasing to meet cryptographic
 requirements.
 
-** Negotiation of Sub SA
+** Negotiation of Sub SAs
+
+Sub SAs are negotiated during the creation of an EESP Child SA, such
+as in the IKE_AUTH or CREATE_CHILD_SA new SA exchanges. To propose
+Sub SA support, the initiator MUST include a USE_EESP_SUB_SA
+notification, which specifies the "Minimum Number of Sub SAs"
+that it supports.
+
+If the responder accepts the request, it MUST include a
+USE_EESP_SUB_SA notification in the response, specifying the
+"Maximum Number of Sub SAs" it supports. The negotiated number
+of Sub SAs for the Child SA MUST be the minimum value between
+these two numbers.
+
+Both peers MUST ensure that the number of Sub SAs established
+complies with the negotiated limit. If at any point a peer attempts
+to exceed the agreed number, the implementation MUST reject such an
+attempt and log an appropriate error.
```

Comment:
How would that attempt look like? An out-of-bounds Session ID?

Reply:
I believe so. If the Session ID is sequential (as we seemed to agree, but now I am not sure :-)), then it is very easy to check.

Reply:
Yes, agreed.